.github/workflows/e2e.yaml

```
@@ -15,7 +15,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Clone repo and checkout submodules
-        uses: actions/checkout@v3.3.0
+        uses: actions/checkout@v3.3.0:
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
```

```suggestion
        uses: actions/checkout@v3.3.0:
        with:
          submodules: recursive
```
That would be easier :)